keyserver/handler.go

```diff
@@ -0,0 +1,15 @@
+package keyserver
+
+import . "github.com/coniks-sys/coniks-go/protocol"
+
+func (server *ConiksServer) handleRegistration(reg *RegistrationRequest) (Response, error) {
+	if len(reg.Username) == 0 || len(reg.Key) == 0 {
```

Comment:
@masomel Do you think we will have some other tasks, related to the server implementation, in these handles?